octobot_trading/producers/ohlcv_updater.py

```diff
@@ -84,7 +85,7 @@ def _create_pair_candle_task(self, pair):
 
                 if candles:
                     last_candle: list = candles[-1]
-                    self.channel.exchange_manager.uniformize_candles_if_necessary(last_candle)
+                    self.channel.exchange_manager.uniformize_candles_if_necessary(candles)
```

Comment:
Is `candles[-1]` modified by `uniformize_candles_if_necessary` correctly update `last_candle` value ?

Reply:
yes

Reply:
OK nice